bundle/config/mutator/python/python_mutator.go

```diff
@@ -103,9 +106,10 @@ func (m *pythonMutator) Apply(ctx context.Context, b *bundle.Bundle) diag.Diagno
 			return dyn.InvalidValue, fmt.Errorf("failed to create cache dir: %w", err)
 		}
 
-		rightRoot, err := m.runPythonMutator(ctx, cacheDir, b.RootPath, pythonPath, leftRoot)
-		if err != nil {
-			return dyn.InvalidValue, err
+		rightRoot, diagnostics := m.runPythonMutator(ctx, cacheDir, b.RootPath, pythonPath, leftRoot)
```

Comment:
Renaming to `diag` will shadow package import for `diag`, I will use `diags` instead.

It doesn't directly use `mutateDiagnostics` because we want both to declare a new variable, and mutate an existing one, and we never want to shadow `mutateDiagnostics`. This can be very error-prone, so we make it very explicit in this case.